game/renderware_binary_stream.ksy

```diff
@@ -54,12 +56,28 @@ types:
             sections::geometry: struct_geometry
             sections::geometry_list: struct_geometry_list
             sections::texture_dictionary: struct_texture_dictionary
+            sections::atomic: struct_atomic
       - id: entries
         type: renderware_binary_stream
         repeat: eos
     instances:
       version:
         value: 'library_id_stamp & 0xFFFF0000 != 0 ? (library_id_stamp >> 14 & 0x3FF00) + 0x30000 | (library_id_stamp >> 16 & 0x3F) : library_id_stamp << 8'
+  struct_atomic:
+    doc-ref: https://gtamods.com/wiki/Atomic_(RW_Section)#Structure
+    seq:
+      - id: frame_index
+        type: u4
+      - id: geometry_index
+        type: u4
+      - id: flag_render
+        type: b1
+      - type: b1
+      - id: flag_collision_test
+        type: b1
+      - type: b24
```

Comment:
These bits must add up to `32`, for example:

```suggestion
      - id: flag_render
        type: b1
      - type: b1
      - id: flag_collision_test
        type: b1
      - type: b29
```

It kind of _works_ even if they don't, but there will be an alignment of the position to a byte boundary just before the following `unused` field, so the 5 bits that you don't parse will be skipped and not accessible in any field.